tests/time_tests/test_runner/conftest.py

```diff
@@ -96,3 +123,43 @@ def get_dict_values(d):
     values = list(get_dict_values(val))
 
     return "-".join(["_".join([key, val]) for key, val in zip(keys, values)])
+
+
+@pytest.mark.hookwrapper
+def pytest_runtest_makereport(item, call):
+    """Pytest hook for report preparation.
+
+    Report tests' data to DB.
```

Comment:
Can we use the words "submit" and "database"? "Make" and "report" is confusing and more to making report document.
Also applicable to the pytest_runtest_makereport function name and --db_report option

Reply:
Docstring was fixed, `--db_report` was renamed to `db_submit` .  `pytest_runtest_makereport ` can't be renamed because it's fixed pytest hook name